platforms/core-runtime/client-services/src/test/groovy/org/gradle/launcher/daemon/client/DaemonClientInputForwarderTest.groovy

```diff
@@ -103,7 +104,7 @@ class DaemonClientInputForwarderTest extends ConcurrentSpecification {
 
     def "one line of text is converted and forwarded as user response"() {
         def event = Stub(PromptOutputEvent)
-        _ * event.convert("def") >> Either.left(12)
+        Specification._ * event.convert("def") >> Either.left(12)
```

Comment:
This looks like an unintentional change.

Reply:
Fixed.